api/sqoop/pom.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>pentaho-big-data-api-sqoop</artifactId><dependencies><dependency><groupId>pentaho-kettle</groupId><artifactId>kettle-core</artifactId><version>6.1-SNAPSHOT</version></dependency></dependencies>
+
```

Comment:
Your pom layout is non-standard-- it might get a frown from larry.  Also, I think kettle dependency versions should use ${project.version}.